File: remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    var l = 0;
    for (var i in nums) {
        if (i == 0 || nums[i] != nums[i-1]) {
            nums[l] = nums[i];
            l += 1;
        }
    }
    return l
};

File: remove-duplicates-from-sorted-array/remove_duplicates_from_sorted_array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        l = 0
        for i in range(len(nums)):
            if i == 0 or nums[i] != nums[i-1]:
                nums[l] = nums[i]
                l += 1
        return l
